cmd/kubectl-testkube/commands/common/uri.go

```diff
@@ -0,0 +1,21 @@
+package common
+
+import "fmt"
+
+func NewCloudUris(rootDomain string) CloudUris {
+	return CloudUris{
+		RootDomain: rootDomain,
+		Api:        fmt.Sprintf("https://api.%s", rootDomain),
+		Agent:      fmt.Sprintf("agent.%s:443", rootDomain),
```

Comment:
is port hardcoded?

Reply:
hmm good question, I'll extract to const for now - as it's usual https port I'll make it configurable when needed (probably never)